### ISSUE TITLE
[Migration] filetime_from_hg

### ISSUE THREAD

**Issue by vishwin** [getpelican/pelican-plugins#1315]
I'd like to migrate this plugin out of this monorepo into the pelican-plugins org, to at the very least serve as a place for continued development and maintenance. I've already [created a repo using the CookieCutter](https://github.com/vishwin/pelican-filetime_from_hg), with the history from here git-filter-branch(1)ed out plus a bugfix commit from me, ready to transfer into the org.

cc: @douardda

**Comment by douardda** [comment]
thanks, looks fine to me, do you need any help from me?

**Comment by kdeldycke** [comment]
@douardda I guess you were CCed because @vishwin identified you as the original author of the plugin.

Looking around I can't unseen the existence of [`filetime_from_git`](https://github.com/getpelican/pelican-plugins/tree/master/filetime_from_git) plugin. @vishwin : what's the effort to merge the two?

**Comment by vishwin** [comment]
They're the same. The entire history from the subdirectory in this monorepo is preserved.

The CC is about requesting some sort of tacit approval from the original author.

**Comment by vishwin** [comment]
My last comment, now deleted, misspoke as I read that as `filetime_from_hg`. `filetime_from_git` is a completely different implementation. This plugin is as simple/short as it is because we use `python-hglib` from the official Mercurial project, and merging with the other plugin would introduce unnecessary complexities, not least of juggling two different library imports. Thus, there is no effort to merge the two.

**Comment by douardda** [comment]
I agree that merging both the plugins would make things unnecessarily complex, even if they share some code (obviously, since `filetime_from_hg` has been written from `filetime_from_git`, as I mentioned in the README file IIRC) .
